proto/poktroll/migration/tx.proto

```diff
@@ -113,24 +113,49 @@ message MsgClaimMorseAccountResponse {
   int64 claimed_at_height = 3;
 }
 
-// TODO_UPNEXT(@bryanchriswhite, #1034):
-// - Refactor to consistent naming convention
-// - Add comments describing the purpose of each message/field
-
+// MsgClaimMorseApplication is used to execute a claim (one-time minting of tokens on Shannon),
+// of the total tokens owned by the given Morse account, according to the on-chain MorseClaimableAccounts,
+// to the balance of the given Shannon account, followed by staking that Shannon account as an application.
 message MsgClaimMorseApplication {
-  option (cosmos.msg.v1.signer) = "shannonDestAddress";
-  string                   shannonDestAddress = 1;
-  string                   morseSrcAddress    = 2;
-  string                   morseSignature     = 3;
-  cosmos.base.v1beta1.Coin stake              = 4 [(gogoproto.nullable) = false];
-  string                   serviceConfig      = 5;
+  option (cosmos.msg.v1.signer) = "shannon_dest_address";
+
+  // The bech32-encoded address of the Shannon account to which the claimed tokens
+  // will be minted and from which the application will be staked.
+  string shannon_dest_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_dest_address"];
+
+  // The hex-encoded address of the Morse account whose balance will be claimed.
+  // E.g.: 00f9900606fa3d5c9179fc0c8513078a53a2073e
+  string morse_src_address = 2 [(gogoproto.jsontag) = "morse_src_address"];
+
+  // The hex-encoded signature, by the Morse account, of this message (where this field is nil).
+  // I.e.: morse_signature = private_key.sign(marshal(MsgClaimMorseAccount{morse_signature: nil, ...}))
+  bytes morse_signature = 3 [(gogoproto.jsontag) = "morse_signature"];
+
+  // The upokt which the Shannon destination account will stake as an application.
+  cosmos.base.v1beta1.Coin stake = 4 [(gogoproto.jsontag) = "stake"];
+
+  // The services this application is staked to request service for.
+  shared.ApplicationServiceConfig service_config = 5 [(gogoproto.jsontag) = "service_config"];
 }
 
+// MsgClaimMorseApplicationResponse is returned from MsgClaimMorseApplication.
+// It indicates the morse_src_address of the account which was claimed, the unstaked
+// balance claimed, the application stake, and the height at which the claim was committed.
 message MsgClaimMorseApplicationResponse {
```

Comment:
I think we should include the stakes application in the response; this would be consistent with the application staking response.